localization/pose_initializer/config/pose_initializer.param.yaml

```
@@ -22,3 +22,7 @@
         0.0, 0.0, 0.0,  0.0,  0.01, 0.0,
         0.0, 0.0, 0.0,  0.0,  0.0,  0.2,
       ]
+
+    # use partial map loading for large maps
+    # Note: You also need to enable partial_map_loading interface in pointcloud_map_loader to use this functionality
+    enable_partial_map_load: true
```

Maybe it was overwritten when adding a comment.
```suggestion
    enable_partial_map_load: false
```

---

Thank you! I resolved here: https://github.com/autowarefoundation/autoware.universe/pull/2500/commits/5dbde23be17426ba393dfe3ba0f8c9e2cbedb2e9